packages/expo-cli/src/commands/eas-build/build/credentials.ts

```diff
@@ -20,7 +21,10 @@ async function ensureCredentialsAutoAsync(
       }
     case Workflow.Generic: {
       const hasLocal = await provider.hasLocalAsync();
-      const hasRemote = await provider.hasRemoteAsync();
+      const hasRemote = skipCredentialsCheck ? false : await provider.hasRemoteAsync();
```

Comment:
`skipCredentialsCheck` should skip validation of credentials, but we still need have those values present.

`ensureCredentials` should not contain any logic related to that flag, for now, only place where we have credentials validation is Ios remote credentials